dm/ctl/common/sql_operation.go

```diff
@@ -20,9 +20,9 @@ import (
 	"github.com/pingcap/dm/dm/command"
 )
 
-// extractBinlogPosSQLPattern extracts binlog-pos and sql-pattern from command
+// ExtractBinlogPosSQLPattern extracts binlog-pos and sql-pattern from command
 // return binlog-pos, sql-pattern, sharding, error
-func extractBinlogPosSQLPattern(cmd *cobra.Command) (string, string, bool, error) {
+func ExtractBinlogPosSQLPattern(cmd *cobra.Command) (string, string, bool, error) {
```

Comment:
should we still put this function into `dm/master`, it seems we wouldn't use it somewhere else, because of `cmd` argument

Reply:
I have the same question, but I did not raise it.:satisfied:

Reply:
@GregoryIan PTAL